src/Blade.php

```diff
@@ -28,6 +32,20 @@ public static function setInstance(BladeInstance $instance)
         static::$instance = $instance;
     }
 
+    /**
+     *  Set the Cache Directory
+     * @param string $dir Path to a custom Cache Directory
+     * @throws \Exception
+     *
+     * @return null
+     */
+    public static function setCacheDir($dir)
```

Comment:
This doesn't set the cache directory does it? It looks like it sets the root we use, the cache directory is beneath it.

Also this library encourages the use of `BladeInstance`, when using this static class you should inject your own blade instance, rather than us supporting every feature of `BladeInstance` in static form here, for example:
```php
$blade = new BladeInstance("/var/my/path", "/tmp/blade");
Blade::setInstance($blade);
```